pyphare/pyphare/pharesee/hierarchy/fromvtkhdf5.py

```
@@ -0,0 +1,431 @@
+                up[i] = box_vals[di + 1]
+            boxes[bi] = Box(lo, up)
+        return boxes
+
+    def __deepcopy__(self, memo):
+        no_copy_keys = ["vtk_file"]  # do not copy these things
+        cpy = phut.deep_copy(self, memo, no_copy_keys)
+        cpy.vtk_file = self.vtk_file
+        return cpy
+
+
+def get_all_available_quantities_from_h5(filepath, time=0, exclude=["tags"], hier=None):
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)